.github/workflows/pr-labeler.yml

```diff
@@ -6,6 +6,6 @@ jobs:
   triage:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/labeler@v3
+    - uses: actions/labeler@v4.0.1
```

Comment:
```suggestion
    - uses: actions/labeler@v4
```

Reply:
Yes, it's better to fix the version to v4

Reply:
We also use v4.0.1 in yari: https://github.com/mdn/yari/blob/9e3b4d482e796b82db6739dc3b9e637748b9530e/.github/workflows/labeler.yml#L9

It's probably better to pin specific versions. Otherwise it's much harder to find out why labeling suddenly starts to fail.